odoo/tools/config.py

```diff
@@ -309,6 +309,9 @@ def __init__(self, fname=None):
                          help="Try to enable the unaccent extension when creating new databases.")
         group.add_option("--geoip-db", dest="geoip_database", my_default='/usr/share/GeoIP/GeoLite2-City.mmdb',
                          help="Absolute path to the GeoIP database file.")
+        group.add_option("--xaccel", dest="xaccel", my_default=False,
+                         help="Enable X-Accel-Redirect content serving from NGINX instead of from Odoo workers.")
```

Comment:
Should the warning be part of the help string, or should I issue it the same we issue deprecation warnings with ``` self._warn_deprecated_options()``` ?

Reply:
I would put it in the help string, so it's visible when people are considering using the feature - plus it's not a deprecation warning (which nobody reads anyway ˆˆ).

It can be terse, e.g. "Requires security precautions, see documentation section XXX" ;)